source/package.json

```diff
@@ -26,7 +26,7 @@
     "compile-ts-stacks": "yarn compile-ts-backend-stack && yarn compile-ts-client-stack",
     "compile-ts-backend-stack": "tsc lib/cdk-textract-stack.ts --target es2018 --module commonjs --allowjs true",
     "compile-ts-client-stack": "tsc lib/cdk-textract-client-stack.ts --target es2018 --module commonjs --allowjs true",
-    "deploy:backend": "ISCICD=$npm_package_cicd ENABLE_KENDRA=$npm_package_enableKendra STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk deploy --toolkit-stack-name DocumentUnderstandingCDKToolkit --require-approval never -v true",
+    "deploy:backend": "ISCICD=$npm_package_cicd ENABLE_KENDRA=$npm_package_enableKendra DEMO_MODE=$npm_package_demoMode READONLY=$npm_package_isROMode STACKNAME=$npm_package_stack_stackname AWS_REGION=$npm_package_stack_region USER_EMAIL=$npm_package_email cdk deploy --toolkit-stack-name DocumentUnderstandingCDKToolkit --require-approval never -v true",
```

Comment:
demoMode should be added as a flag in package.json

Reply:
And we need to add it in the CICD deploy as well : https://github.com/awslabs/document-understanding-solution/blob/kendraMaster/deployment/document-understanding-solution.template#L28

Reply:
re including in package.json - ah yes my apologies - it got excluded from the commit because I was avoiding committing unnecessary flag changes.

For CICD deployment - sure, will add.